src/app/tests/suites/certification/Test_TC_MF_1_30.yaml

```diff
@@ -0,0 +1,116 @@
+# Copyright (c) 2021 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name:
+    22.1.30. [TC-MF-1.30] Open commissioning window - normal operation and
+    closing [ECM] [DUT - Commissionee]
+
+config:
+    nodeId: 0x12344321
+    timeout: 330
+    endpoint: 0
+
+tests:
+    - label: "Factory Reset the DUT_CE"
+      cluster: "SystemCommands"
+      command: "FactoryReset"
+
+    - label: "TH_CR1 starts a commissioning process with DUT_CE"
+      cluster: "DelayCommands"
+      command: "WaitForCommissionee"
+      arguments:
+          values:
+              - name: "nodeId"
+                value: nodeId
+
+    - label: "TH_CR1 opens a commissioning window on DUT_CE"
+      cluster: "AdministratorCommissioning"
+      command: "OpenCommissioningWindow"
+      timedInteractionTimeoutMs: 10000
+      arguments:
+          values:
+              - name: "CommissioningTimeout"
+                value: 300
```

Comment:
This can be much shorter for the purposes of the test. It could literally be 10 seconds in total, since the TH_CR1 is still connected to the node, and can read the window status beforehand, open window, wait a bit, and then check window closed.

Reply:
You can't open a window for shorter than 180s, per spec.  But yes, it could be 180s.....

Reply:
@tcarmelveilleux i am updating the value as per the testplan
![image](https://user-images.githubusercontent.com/102359958/165800912-9484be4e-ccda-4601-9b73-7d7184c9ce29.png)